packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs'
+import path from 'path'
+
+import execa from 'execa'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+export const command = 'run side'
+export const description = 'Run server for api in production'
+export const builder = (yargs) => {
+  yargs
+    .positional('side', {
```

Comment:
I'd skip "side" for now unless it adds necessary clarity. For me, it begged the question "can I run something other than API?"

Again, not a strong preference.

Reply:
So I added side because by default we have the two sides, and it's more explicit that I'm starting/running a server for only the api